lib/deimos/utils/db_poller/state_based.rb

```diff
@@ -26,12 +26,16 @@ def process_updates
             success = process_batch_with_span(batch, status)
             finalize_batch(batch, success)
           end
-          Deimos.config.logger.info("Poll #{@producer.topic} complete (#{status.report}")
+          Deimos.config.logger.info("Poll #{@producers.map(&:topic)} complete (#{status.report}")
```

Comment:
This probably should be a method on the base poller class e.g.

```
def log_identifier
  "#{self.class.name}: #{@producers.map(&:topic)}"
end  
```

Whatever we end up doing with the aforementioned resource_class can help this.